src/client/grpc_client.erl

```diff
@@ -552,6 +556,9 @@ stream_handle({gun_data, _GunPid, _StreamRef, fin, Data},
             handle_remote_closed([], Stream#{recvbuff => <<>>, mqueue => MQueue ++ Frames})
     end;
 
+stream_handle({gun_error, _GunPid, _StreamRef, {stream_error, no_error, 'Stream reset by server.'}},
```

Comment:
Does `'Stream reset by server.'` indeed come as an atom?  :see_no_evil:

Reply:
Yes, it's gun's constant